airflow/models.py

```diff
@@ -48,7 +49,7 @@
 from sqlalchemy import case, func, or_, and_
 from sqlalchemy.ext.declarative import declarative_base, declared_attr
 from sqlalchemy.dialects.mysql import LONGTEXT
-from sqlalchemy.orm import relationship, synonym
+from sqlalchemy.orm import reconstructor, relationship, synonym
```

Comment:
first time I see `@reconstructor`, SQLalchemy is one of these bottomless libs that keep on giving